### PR TITLE
Add express-minify.

### LINKED ISSUE #744: Minify our assets

### PR DESCRIPTION
**Pull request by XhmikosR**
Basically reverts commit ab439e7d88d444dae7d3b8447eb2d21bf3a09d25.

Doesn't seem to work though, the SRI hashes don't match.

Fixes #744.

### CODE REVIEW

**Comment by jmervine** [comment]
Because the files are run through [`generateSRI`](https://github.com/MaxCDN/bootstrap-cdn/blob/develop/routes/index.js#L24-L30), the SRI should be right...

**Comment by XhmikosR** [comment]
Must be an issue with the order. I can't think of anything else.

File: app.js [hunk 1]
```diff
@@ -54,6 +55,7 @@ if (env === 'production') {
 
 // middleware
 app.use(compression());
+app.use(minify());
```
Comment (jmervine) [inline app.js]: I believe this should be above `compression()`.

Comment (XhmikosR) [inline app.js]: I just followed their docs https://github.com/SummerWish/express-minify#basic-usage

Comment (jmervine) [inline app.js]: I see, it was just a guess.

**Comment by jmervine** [comment]
@XhmikosR Did the revert fix the SRI issue?

**Comment by XhmikosR** [comment]
Nope this is still broken.

On a side note, it seems we don't use any compression for static assets like CSS and JS files.

**Pull request closed without merging**